### PR TITLE
PAINTROID-554 Adjust Colorwheel in Pocket Paint

### PR DESCRIPTION
**Pull request by foenabua**
https://jira.catrob.at/browse/PAINTROID-554

Use xml file instead of png for Color Wheel.
Additionally updated UIMatcher to work with Vector Drawables and also discovered  and fixed one copy&paste error in UIMatcher.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by Lenkomotive — APPROVED** [review]
👍🏼

**Review by Lenkomotive — CHANGES_REQUESTED** [review]
Please checkout the last build in jenkins for this branch. It seems that exchanging the png with xml also requires some adjustments in some tests (ColorDialogIntegrationTest). Please also make sure that similar issues are not present in the implementation.

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/UiMatcher.kt [hunk 1]
```diff
@@ -249,13 +257,22 @@ object UiMatcher {
...
+
                 if (targetDrawable is BitmapDrawable) {
-                    val bitmap: Bitmap = targetDrawable.bitmap
-                    return bitmap.sameAs(expectedBitmap)
+                    val targetBitmap: Bitmap = targetDrawable.bitmap
+                    val expectedBitmap = (expectedDrawable as BitmapDrawable).bitmap
+                    return targetBitmap.sameAs(expectedBitmap)
+                } else if (targetDrawable is VectorDrawable) {
+                    val targetBitmap: Bitmap = vectorToBitmap(expectedDrawable as VectorDrawable)
+                    val expectedBitmap = vectorToBitmap(targetDrawable)
```
Comment (Lenkomotive) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/UiMatcher.kt:270]: Looks like target and expected are mixed up here.

**Review by Lenkomotive — CHANGES_REQUESTED** [review]

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/UiMatcher.kt (hunk 1 above)
Comment (foenabua) [reply Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/UiMatcher.kt]: Has been fixed!

**Review by Lenkomotive — APPROVED** [review]
👍🏼

**Review by SebastianGrief — APPROVED** [review]
LGTM 👍

**Pull request merged**